Flutter/lib/DarkThemePref.dart

```diff
@@ -0,0 +1,15 @@
+import 'import_all.dart';
+
+class DarkThemePreference {
```

Comment:
Can DarkThemePreference and DarkThemeProvider go in AppTheme? I don't see why they need their own files

Reply:
Resolved in commit `app-style 43887b4`